assets/src/edit-story/components/keyboard/index.js

```diff
@@ -87,7 +87,7 @@ function useKeyEffectInternal(
       const handler = createKeyHandler(node, keySpec, batchingCallback);
       mousetrap.bind(keySpec.key, handler, type);
       return () => {
-        mousetrap.unbind(keySpec.key);
+        mousetrap.unbind(keySpec.key, type);
```

Comment:
Without this, the handler in line 88 isn’t actually un-bound. When we cut before, it would un-mount the element, but not remove the callback. Therefore pressing the meta key would actually call a setState on an un-mounted element thus raising an exception. This fixes that.